conf/agent/agent.conf

```diff
@@ -26,4 +26,11 @@ plugins {
         plugin_data {
         }
     }
+    WorkloadAttestor "docker" {
+        plugin_data {
+            cgroup_container_index = 3
+            cgroup_prefix = ""
+            labels = [""]
```

Comment:
I think we can remove `labels` and possibly the other two as well if we decide to leave them undocumented